src/components/socialMediaBar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import "./socialMediaBar.scss"
+
+export default function SocialMediaBar() {
+  return (
+    <div className="social-media-bar">
+      <ul>
+        <a href="https://www.facebook.com">
+          <li id="fb-logo" className="social-media-bar__image"></li>
+        </a>
+        <a href="https://www.twitter.com">
+          <li id="twitter-logo" className="social-media-bar__image"></li>
+        </a>
+        <a href="https://www.gmail.com">
+          <li id="email-logo" className="social-media-bar__image"></li>
+        </a>
+      </ul>
+    </div>
```

Comment:
I'm not really sure how useful this whole thing is. Ideally we want to change this into buttons allowing someone to share the current news story.

Take a look at how Facebook formats share links: https://developers.facebook.com/docs/plugins/share-button/

We can mimic this using the current page url, and follow something similar for Twitter. Email should really be using "mailto: email@address?subject=<page-name>" or something along these lines.

I know this is how it is done on production, but I think this is a worthwhile and small improvement we can go ahead and make.

You can also search for some existing Gatsby plugin to handle social media sharing, and leverage that.

Reply:
I agree that would be a nice improvement, the current social media bar isn't very helpful at the moment. I'll look into updating that to share the current news article on FB/Twitter/email.